family_plus/website/templates/home_page/welcome.html

```diff
@@ -1,25 +1,104 @@
-{% extends 'header_footer/base.html' %}
-
-{% block title %}
-  Welcome | Family+
-{% endblock %}
-
-{% block content %}
-
-<body>
-  <section style="padding-top: 70px; min-height: 1000px; padding-bottom: 100px;">
-    <center>
-      <div class="container-fluid">
-        <h4>Welcome Page Creation In Progress...</h4>
-      </div>
-      <div class="container-fluid" style="padding-top: 30px;">
-        <a href="{% url 'home' %}" class="btn btn-secondary">
-          Home
-        </a>
-      </div>
-    </center>
-  </section>
-</body>
-
-{% endblock %}
 
+<!doctype html>
+<html lang="en">
+    <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+     <br>
+    <title>Family+ Welcome Page</title>
```

Comment:
You are missing the code for extending base.html, the block title, and block content. From line 1 to 10 are already included in base.html.